torchvision/prototype/transforms/functional/_color.py

```diff
@@ -435,7 +435,14 @@ def equalize(inpt: features.InputTypeJIT) -> features.InputTypeJIT:
         return equalize_image_pil(inpt)
 
 
-invert_image_tensor = _FT.invert
+def invert_image_tensor(image: torch.Tensor):
+    num_channels, height, width = get_dimensions_image_tensor(image)
+    if num_channels not in (1, 3):
+        raise TypeError(f"Input image tensor can have 1 or 3 channels, but found {num_channels}")
```

Comment:
Not sure why we have this check since the number of channels is irrelevant here.

Reply:
Yes, makes sense to remove that, just check for input as tensor ?

Reply:
I got a similar situation here: https://github.com/pytorch/vision/pull/6820/files#r1003210245

Should I remove it?